packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -662,6 +662,7 @@ const VirtualListBaseFactory = (type) => {
 			if (getDirection(keyCode)) {
 				ev.preventDefault();
 				this.setSpotlightContainerRestrict(keyCode, target);
+				Spotlight.setPointerMode(false);
```

Comment:
@webOS101 
Do you agree to call `Spotlight.setPointerMode(false)` in `VirtualList? You did not recommend to do it before. We did not consider it as our solution.

Reply:
Generally, that's true. We should avoid manually setting pointer mode to allow Spotlight to manage it according to its rules. However in this case, we _may_ be preventing Spotlight's management by stopping event propagation. There's likely some API refinement we should do to better handle this but, in the mean time, this approach within the framework components is usually acceptable.

@germboy - In that light, should we move the call to `setPointerMode()` into the `if` block below to only set it in the case that we'd be preventing Spotlight management?

Reply:
@ryanjduffy
Thanks for clarifying it.